unittests/test_data/image_manifest.yaml

```diff
@@ -0,0 +1,14 @@
+manifest:
```

Comment:
While this test data does work to get the unit tests running, the actual data that's obtained while inspecting the image is in the JSON format. I think it would be better to use actual examples that were obtained by inspecting an example image. @hongweiliu17 I'll make sure to send them to you so you can use them here.

Reply:
Thanks. I convert the the two yaml files to json files and will look for the actual examples by inspecting an image.
Actually I can find json files for other sanity test, except the clair data. Could you send me a Clair data example?

Reply:
Updated. Could you take a look again?